master/pkg/model/experiment.go

```diff
@@ -53,7 +53,7 @@ const (
 	// DeletingState constant.
 	DeletingState State = "DELETING"
 	// DeleteFailedState constant.
-	DeleteFailedState State = "DELETE_ERROR"
+	DeleteFailedState State = "DELETE_FAILED"
```

Comment:
pulled this out https://github.com/determined-ai/determined/pull/3211, nothing can be wrong since it's a mismatch with the postgres enum (inserts just failed 😟 )

Reply:
https://determinedai.atlassian.net/browse/DET-6295